k8_cortx_cloud/status-cortx-cloud.sh

```diff
@@ -115,6 +115,7 @@ fi
 
 # Check pods
 expected_count=$(helm get values cortx --all --namespace "${namespace}" --output yaml | yq .control.replicaCount)
+[[ ${data_deployment} == true ]] && expected_count=0
```

Comment:
Another approach would be to check the`.control.enabled` value, but I suppose solution.yaml should take precedence for the scripts.